libkpod/config.go

```diff
@@ -134,6 +134,12 @@ type RuntimeConfig struct {
 	// handle cgroups for containers.
 	CgroupManager string `toml:"cgroup_manager"`
 
+	// HooksDirPath location of oci hooks config files
+	HooksDirPath string `toml:"hooks_dir_path"`
+
+	// Hooks List of hooks to run with container
+	Hooks map[string]HookParams
```

Comment:
This isn't in the TOML config, so can we not have it in the config structure? Maybe in ContainerServer instead?

Reply:
Fixed

Reply:
Moved